src/go/migration-cli/cmd/root.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	vectorizedv1alpha1 "github.com/redpanda-data/redpanda/src/go/k8s/apis/vectorized/v1alpha1"
+	"github.com/redpanda-data/redpanda/src/go/migration-cli/cmd/version"
+	"github.com/redpanda-data/redpanda/src/go/migration-cli/pkg/migration"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/i18n"
+
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	defaultName            = "redpanda-cluster"
+	defaultNamespace       = "default"
+	DefaultChartRefVersion = "4.0.54"
+)
+
+type RootFlags struct {
+	ClusterFile string
+	ConsoleFile string
+	OutputFile  string
+
+	ChartVersion              string
+	RedpandaImageTag          string
+	RedpandaContainerRegistry string
+}
+
+func Execute() error {
+	flags := &RootFlags{}
+
+	// RootCmd represents the base command when called without any subcommands
+	root := &cobra.Command{
+		Use:   "migration-cli",
```

Comment:
no, too generic. i'd say 
`k8s-v1-v2-operator-migration`  or smth like that since this will be opefully only executed once, we should err on the side of clarity not CLI ergonomics.

Reply:
We should not use v1/v2 in the naming. Call it what it is, `cluster-to-redpanda-migration`